marlowe-dashboard-client/src/MainFrame/State.purs

```diff
@@ -86,12 +86,23 @@ handleQuery ::
   Query a -> HalogenM State Action ChildSlots Msg m (Maybe a)
 handleQuery (ReceiveWebSocketMessage msg next) = do
   case msg of
-    WS.WebSocketOpen -> assign _webSocketStatus WebSocketOpen
+    WS.WebSocketOpen -> do
+      assign _webSocketStatus WebSocketOpen
+    {- FIXME: uncomment this when the websocket communication is fixed
+      -- potentially renew websocket subscriptions
+      mPlayState <- peruse _playState
+      for_ mPlayState \playState -> do
+        let
+          wallet = view (_walletDetails <<< _walletInfo <<< _wallet) playState
+
+          followAppIds :: Array PlutusAppId
+          followAppIds = Set.toUnfoldable $ keys $ view _allContracts playState
+        subscribeToWallet wallet
+        for followAppIds subscribeToPlutusApp -}
     (WS.WebSocketClosed closeEvent) -> do
-      -- TODO: When we change our websocket subscriptions, the websocket is closed and then reopened.
-      -- We don't want to warn the user in this case, but we do want to warn them if the websocket
-      -- closes for some unexpected reason. But currently I don't know how to distinguish these cases.
-      -- addToast $ connectivityErrorToast "Connection to the server has been lost"
+      -- TODO: Consider whether we should show an error/warning when this happens. It might be more
+      -- confusing than helpful, since the websocket is automatically reopened if it closes for any
+      -- reason.
```

Comment:
eventually we could show an error if it can't reconnect N-tries in a row, for now I'd log it if there isn't already an error in the console. If there's already a default browser error I'd do nothing